docs/references/expo/custom-auth-flows/expo-impersonation.mdx

```diff
@@ -0,0 +1,491 @@
+---
+description: Learn how to build custom Impersonation into your Expo application
+---
+
+# Impersonation in Expo
+
+<Callout type="warning">
+The following flows are built assuming your Clerk app is using the default instance settings. If you have changed your instance settings, you may need to make changes to the examples.
+</Callout>
+
+User Impersonation is a powerful feature to support your customers with. Currently, from the Clerk Dashboard, [you can impersonate your users](https://clerk.com/docs/custom-flows/user-impersonation#impersonate-a-user-from-the-clerk-dashboard) and see through their eyes inside of your web applications. In this example, we’ll learn how to build impersonation into your mobile application through [Clerk’s custom flows](https://clerk.com/docs/custom-flows/overview). 
+
+The example we’ll build will still need some work before it goes into your app. For example, we will build impersonation directly into the dashboard, whereas you should build your impersonation into an admin only dashboard, where all of your users are listed & can be interacted with. You will want to restrict this so **only employees of your company or those who should have access to other user's information** can use this implementation.
+
+## Before you start
+
+Before continuing, please go through the [Expo Quickstart](https://clerk.com/docs/quickstarts/expo) and utilize the repo as a starting point for this example.
+
+```bash
+git clone https://github.com/clerk/clerk-expo-quickstart
+```
```

Comment:
````suggestion
1. Follow [Clerk's Expo quickstart](/docs/quickstarts/expo) to ensure your project is configured properly.
````

I'm glad you added this point about them needing to follow the quickstart. I'll add it to the other expo custom flows in this PR 👍.

I cut the line about cloning the repo because we don't tend to point people to example repos outside of the quickstarts (as far as I can tell). I also personally think if cloning the repo isn't _required_, best to cut it so the guide only has everything they absolutely need